deploy/complete/helm-chart/mushop/charts/fulfillment/values.yaml

```diff
@@ -0,0 +1,18 @@
+replicaCount: 1
+
+image:
+  repository: iad.ocir.io/oracle/ateam/mushop-fulfillment
+  tag: 1.0.0
+  pullPolicy: IfNotPresent
+
+osb: {}
+
+env:
+  natsHost: "nats"
+  natsPort: 4222
+  newOrdersSubject: mushop-orders
+  shippedOrdersSubject: mushop-shipments
+  simulationDelay: 10000
+
+service:
+  port: 80
```

Comment:
80 or 8099?

Reply:
8099 is what the app listens on. not really relevant. 
80 is whats mapped elsewhere.  values.service.port is used in the template as the service port, and the container port is 8099.

Reply:
Not relevant, just odd.